ngi_pipeline/conductor/launchers.py

```diff
@@ -66,7 +76,31 @@ def launch_analysis(projects_to_analyze, restart_failed_jobs=False,
         except (RuntimeError, KeyError, CharonError) as e: # BPA missing from Charon?
             LOG.error('Skipping project "{}" because of error: {}'.format(project, e))
             continue
+        try:
```

Comment:
this try should always work.... we run QC for all samples that get processed... this means also Uppsala samples.... this is not bad but only to be sure about this...